airbyte-ci/connectors/connector_ops/connector_ops/required_reviewer_checks.py

```diff
@@ -54,14 +55,17 @@ def find_mandatory_reviewers() -> List[Dict[str, Union[str, Dict[str, List]]]]:
             "teams": list(BREAKING_CHANGE_REVIEWERS),
             "is_required": utils.get_changed_metadata(diff_regex="upgradeDeadline"),
         },
+        {
+            "name": "Build customization",
+            "teams": list(BUILD_CUSTOMIZATION_REVIEWERS),
+            "is_required": utils.get_changed_file(utils.BUILD_CUSTOMIZATION_FILE_NAME),
+        },
```

Comment:
this seems like it would work, but is there a reason we're doing it here instead of codeowners? We need to have these more dynamic checks because we're checking the file contents or basing it off of some more complex criteria like metadata info, but I think when it's just a direct file change codeowners can over it?

What do you think?